test/__snapshots__/transform.spec.js.snap

```diff
@@ -29,3 +29,12 @@ Object.defineProperty(exports, 'URL', {
 exports.foo = foo;
 "
 `;
+
+exports[`process should transform relative imports 1`] = `
+"'use strict';
+
+const foo = 42;
```

Comment:
Bundled code will break the code coverage, imported files should remain external.

Reply:
Yes, hence my question ;)

Reply:
Also you transpile the config file, there it definitely makes sense to include all imports

Reply:
I implemented now that by default, no imports are resolved, except for the config, where relative imports are resolved. It is controlled by an option.